### ISSUE TITLE
Intersect Town Hall - Thursday, 26th September 2024

### ISSUE THREAD

**Issue by whitevo** [Catalyst-Swarm/Intersect-Town-Hall#68]
# Town Hall agenda and planning
- https://docs.google.com/document/d/17bGmeZ8F4-TKMv_lN0mE99xyqfpASdKd-9kHk6OVoh8

# Slides
- https://docs.google.com/presentation/d/1JIC0HrFnG7qUNJrvuTf21rU4uPC5VbsrPphr9I8DhSs

# Facilitators
- Tevo, Vani

# Video
- https://www.youtube.com/watch?v=Pv8FDAKKfCo

# Meeting Summary Doc
- https://docs.google.com/document/d/1ZMvzYAD2SUM7ptg4bSusHaO-zOgHYpgOFs-DZyUIWNM

# Actions
- [x] Add Agenda Preparation Doc
- [x] Include Slides
- [x] Include Facilitators
- [x] Add Video
- [x] Add Meeting Summary
- [x] Remind Intersect Staff to include Description to their YouTube videos and provide source material

# Other Related Documents
Sheet for collecting timestamps
- https://docs.google.com/spreadsheets/d/1VCAQDpT3vKrqf9MvwoApw5Z9TMkrtiM5PYaK_rfVolg/edit?gid=0#gid=0

> The same source will be used for both TH and ATH timestamps

**Comment by whitevo** [comment]
Until today video still not done uploaded. Gave 3rd and final reminder today

**Comment by whitevo** [comment]
More organised town hall as templates are now more refined and don't need many edits anymore.

There still continue issues with Town Hall amplification in Discord and getting information into the Newsletter.

we started testing Zoom topics. where most voted topics get to be discussed after Human Interfacing section.

We made Human Interfacing sessions longer, which was well received, and time feels more right.

More additions on setting context for Intersect groups like its structure and core focus
